### DISCUSSION TITLE
Black screen after the app startup

### DISCUSSION THREAD

**Discussion by bcliutong** [airsdk/Adobe-Runtime-Support#2471] [Q&A]
I set the app to boot automatically, but sometimes the app stays on the black interface after starting (normally, the black interface is displayed first, then the stage color is displayed, and finally the elements in the app are displayed)

**Comment by itlancer** [comment]
Does this happens on Android?
I can obsereve multiple issues with black screen for Android:
https://github.com/airsdk/Adobe-Runtime-Support/issues/2396
https://github.com/airsdk/Adobe-Runtime-Support/issues/2017
https://github.com/airsdk/Adobe-Runtime-Support/issues/1194
https://github.com/airsdk/Adobe-Runtime-Support/issues/1003
https://github.com/airsdk/Adobe-Runtime-Support/issues/698
https://github.com/airsdk/Adobe-Runtime-Support/issues/151
https://github.com/airsdk/Adobe-Runtime-Support/issues/6
